RELEASE.md

```diff
@@ -1,3 +1,285 @@
+# Release 2.0.0-rc0
+
+## Major Features and Improvements
```

Comment:
We should link to the migration instructions, converter script, etc, very soon here

Reply:
It does link to those guides. Do you mean link directly to the upgrade script?

Reply:
It does, but not right at the top.